frontend/public/components/cluster-service-class.tsx

```diff
@@ -1,16 +1,17 @@
 /* eslint-disable no-undef */
 
 import * as React from 'react';
-import { Link } from 'react-router-dom';
+import * as _ from 'lodash-es';
 
+import { Link } from 'react-router-dom';
 import { ColHead, DetailsPage, List, ListHeader, ListPage, ResourceRow } from './factory';
-import { SectionHeading, detailsPage, navFactory, ResourceSummary, resourcePathFromModel } from './utils';
+import { SectionHeading, detailsPage, navFactory, ResourceSummary, resourcePathFromModel, ResourceLink } from './utils';
 import { viewYamlComponent } from './utils/vertnav';
 import * as classNames from 'classnames';
```

Comment:
And this on line 6?

Reply:
Ok, can do.  What are you basing this on?  Not sure what the rules are for listing the imports.  -thanks

Reply:
It's imports for dependencies, then a space, then imports from the console itself

Reply:
fixed